packages/bff/src/services/catalogService.ts

```diff
@@ -174,21 +159,18 @@ const getAttributeIds = (
   ),
 ];
 
-export const fetchAllEserviceConsumers = async (
+export const getAllEserviceConsumers = async (
   catalogProcessClient: CatalogProcessClient,
   headers: Headers,
   eServiceId: EServiceId
 ): Promise<catalogApi.EServiceConsumer[]> =>
-  await getAllFromPaginated(async (offset: number) =>
+  await getAllFromPaginated(async (offset, limit) =>
     catalogProcessClient.getEServiceConsumers({
       headers,
       params: {
         eServiceId,
       },
-      queries: {
-        offset,
-        limit: 50,
```

Comment:
No need to set this to 50, `getAllFromPaginated` already does that when calling this paginated callback